clusterloader2/testing/load/config.yaml

```diff
@@ -20,6 +20,7 @@
 {{$ENABLE_PROMETHEUS_API_RESPONSIVENESS := DefaultParam .ENABLE_PROMETHEUS_API_RESPONSIVENESS false}}
 {{$ENABLE_PVS := DefaultParam .CL2_ENABLE_PVS true}}
 {{$ENABLE_NETWORKPOLICIES := DefaultParam .CL2_ENABLE_NETWORKPOLICIES false}}
+{{$ENABLE_DNSTESTS := DefaultParam .CL2_ENABLE_DNSTESTS false}}
```

Comment:
Can you change that to true for now to test that behavior (and once done job with the test in the PR itself).

Reply:
It used to be true in this PR, but I asked to revert before submitting this PR.

Given that we are going to change this PR anyway I suggest to:
1. Applying comments to this PR and changing this to true
2. Waiting for prow to trigger presubmit
3. Changing this to false

This way we will test this PR with the changes needed to apply comments and ultimately submit this without enabling everywhere.

Reply:
That's exactly what I was suggesting.

Reply:
I had the bool set to true and one round of presubmit run over the weekend. I have set it to true again after addressing all the other comments.